src/scripts/Player.ts

```diff
@@ -18,6 +18,8 @@ class Player {
     private _shinyList: boolean[];
     private _notCaughtBallSelection: GameConstants.Pokeball;
     private _alreadyCaughtBallSelection: GameConstants.Pokeball;
+    private _sortOption: KnockoutObservable<string>;
```

Comment:
I missed out some "", it should just be the string which is displayed in the options list